lib/vmpooler/api/v1.rb

```diff
@@ -892,6 +892,22 @@ def invalid_pool(payload)
             when 'lifetime'
               need_token! if Vmpooler::API.settings.config[:auth]
 
+              # in hours, defaults to one week
+              max_lifetime_upper_limit = config[:max_lifetime_upper_limit]
```

Comment:
Ah, I found the issue. This key is a string, not a symbol. When I switched that out, it works.

Reply:
good catch. That was an issue with testing this as well, since we set the parameters and I wasn't sure if it was a symbol or a string. Thanks!